app/infrastructure/database/migrations/001_jobs_create.rb

```diff
@@ -8,7 +8,7 @@
       # Method name # Column
       primary_key :id
 
-      Integer     :job_id, unique: true
+      String      :job_id, unique: true
```

Comment:
I'm not sure why should `job_id` be a string, but I'll pass it for now.

Reply:
because `job_id` in the `PartialJob` is [defined as a string](https://github.com/KeroKnows/s-killer/pull/13/files#diff-41263f840fc2b0e55129d70c9505b7baaa602203eae3a8887455789dab82b342R9)